Sprint-3/revise/implement/repeat.test.js

```diff
@@ -23,3 +23,13 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
+function repeatStr(str, count){
+    if(count < 0){
+        return "Error: Count cannot be negative"
```

Comment:
This function should "throw an error" instead of "return an error" when `count < 0`.
May I suggest use Google/ChatGPT to find out more about "How to throw an error in JS?" (You may also want to ask "How to catch a thrown error in JS?"

Reply:
I have implemented the changes